### PR TITLE
fix: mui textfield disappearing

### PR DESCRIPTION
**Pull request by patrickjmeurer**
Temporary fix for #90  !

:)

### CODE REVIEW

**Comment by eduardoborges** [comment]
Hey @patrickjmeurer! Can you update package lock file?

And can you provide some evidences? screenshots or videos of your fix.

**Comment by patrickjmeurer** [comment]
Hello @eduardoborges. As you requested, in annex the video of the solution, with the packages already updated!

https://github.com/user-attachments/assets/d763d6bc-4289-4ae7-a18e-6d594de8c89b

**Comment by patrickjmeurer** [comment]
idk why the CI is failing, maybe cache?

**Review by regivaldo — APPROVED** [review]

**Review by regivaldo — APPROVED** [review]

**Comment by patrickjmeurer** [comment]
@eduardoborges can I help with anything else on this PR?

**Comment by eduardoborges** [comment]
> @eduardoborges can I help with anything else on this PR?

This version of inputmask doesnt exists

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "react-dom": ">=16.4 || 17 || 18 || 20"
   },
   "devDependencies": {
-    "inputmask": "5.0.10-beta.9",
+    "inputmask": "5.0.10-beta.11",
```
Comment (patrickjmeurer) [inline package.json:39]: @eduardoborges I’ve updated the package to the latest beta version.

**Comment by patrickjmeurer** [comment]
@eduardoborges some news?

**Comment by eduardoborges** [comment]
@patrickjmeurer Can you solve the conflicts?

**Comment by patrickjmeurer** [comment]
@eduardoborges done! :)

**Review by eduardoborges — APPROVED** [review]

**Pull request merged**